.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
```

Comment:
So, here's the thing: Mezzio itself supports 5.6 and 7.0 in the v2, but drops them in v3.
I'm not able to build a test playground for v2 thou because mezzio-skeleton only starts with v3.
Here I chose to drop PHP 5.6,7.0 and Mezzio v2 support to ease the maintenance of the package, but this should be pointed out in the doc, as Codeception as a whole supportes 5.6 and 7.0 instead.

Reply:
Who cares about Mezzio v2?
It is Zend Expressive v2, and people who are still using it don't care much about upgrades, right?